akka-cluster-sharding/src/main/scala/akka/cluster/sharding/WowAllocationStrategy.scala

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.cluster.sharding
+
+import scala.collection.immutable
+import scala.concurrent.Future
+
+import akka.actor.ActorRef
+import akka.cluster.sharding.ShardCoordinator.ShardAllocationStrategy
+import akka.cluster.sharding.ShardRegion.ShardId
+
+object WowAllocationStrategy {
+  private val emptyRebalanceResult = Future.successful(Set.empty[ShardId])
+}
+
+// FIXME name
```

Comment:
We will probably just call this LeastShardAllocationStrategy2, and it will eventually become the default and the old will be phased out.